crates/forge_main/src/banner.rs

```diff
@@ -1,10 +1,24 @@
 use std::io;
+use std::path::Path;
 
 use colored::Colorize;
+use dirs::home_dir;
+use dotenv::dotenv;
+use forge_display::title::TitleFormat;
 
 const BANNER: &str = include_str!("banner");
 
 pub fn display() -> io::Result<()> {
+    let dotenv_path = dotenv().ok();
```

Comment:
We read the env already in env.rs. We shouldn't read it here again. We should store that information in `Environment` and then use that.